.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -47,20 +52,3 @@ Choose one of the following:
 - [ ] I have opened a PR in the [docs repo](https://github.com/freedomofpress/securedrop-docs) for these changes, or will do so later
 - [ ] I would appreciate help with the documentation
 - [ ] These changes do not require documentation
-
-### If you added or updated a production code dependency:
```

Comment:
It might make sense to restore this section and retitle it something like "If you added/updated a reference to a production code dependency", and then have a clause like "If you silenced an alert related to a dependency, please provide a justification for doing so for review purposes:"